archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/CodeUnitsShould.java

```diff
@@ -69,6 +69,19 @@
     @PublicAPI(usage = ACCESS)
     CONJUNCTION notHaveRawParameterTypes(Class<?>... parameterTypes);
 
+    /**
+     * Asserts that only certain objects call the code units selected by this rule.
+     * <br><br>
+     * E.g.
+     * <pre><code>
+     * {@link ArchRuleDefinition#codeUnits() codeUnits()}.{@link GivenCodeUnits#should() should()}.{@link CodeUnitsShould#onlyBeCalled()} onlyBeCalled().{@link OnlyBeCalledSpecification#byClassesThat(DescribedPredicate)}  byClassesThat(equivalentTo(SomeClass.class))}
```

Comment:
Instead of the version with predicate `byClassesThat(equivalentTo(SomeClass.class))` maybe we can give an example with the fluent API, e.g. `byClassesThat().areAnnotatedWith(SomeAnnotation.class)` or something like that? :thinking:

Reply:
I've used example from tests.